Dockerfile.ubi

```diff
@@ -1,5 +1,5 @@
 # Conjur Base Image (UBI)
-FROM cyberark/ubi-ruby-fips:latest
+FROM registry2.itci.conjur.net/cyberark/ubi-ruby-fips:ubi8-b201fa9
```

Comment:
Should be switched back after base image release

Reply:
Will rollback before push to master

Reply:
Fixed, pushing to master